maintainers/upload-release.pl

```diff
@@ -171,6 +171,10 @@ sub downloadFile {
     downloadFile("binaryTarballCross.x86_64-linux.armv7l-unknown-linux-gnueabihf", "1");
 };
 warn "$@" if $@;
+eval {
+    downloadFile("binaryTarballCross.x86_64-linux.riscv64-linux-gnu", "1");
```

Comment:
Forgot the `-unknown-`
```suggestion
   downloadFile("binaryTarballCross.x86_64-linux.riscv64-unknown-linux-gnu", "1");
```